docs/swagger.yml

```diff
@@ -580,6 +580,8 @@ definitions:
         type: "array"
         items:
           $ref: "#/definitions/SuiteOption"
+      suite_summary:
```

Comment:
💯 for remembering to update the swagger, we always forget that the first time around :)